io.github.sil_car.SqueezeVid.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Name=Squeeze Vid
+NoDisplay=true
+Comment=CLI app
```

Comment:
This is incorrect. The space says:

>  Tooltip for the entry, for example "View sites on the Internet". The value should not be redundant with the values of Name and GenericName. 

So "CLI app" is what the application is.